### PR TITLE
 API disparity between GraphQL and REST

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

**Comment by karensawrey** [comment]
@plaindocs this is by no means a final version, but please take a look.

File: pages/api_differences.md.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+* Buildkite's terraform provider uses GraphQL where it can, but it falls back to using REST for setting [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
+* REST API can [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [`buildkite-agent annotate` command](/docs/agent/v3/cli-pipeline) from within a job).
+* [Provider setting properties](/docs/apis/rest-api/pipelines#provider-settings-properties) `provider_settings` allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on `pipeline create`.
+* REST API supports [job logs](/docs/apis/rest-api/jobs#get-a-jobs-log-output).
+* [Getting a list of IPs from which Buildkite sends webhooks](https://buildkite.com/docs/apis/rest-api/meta#get-meta-information).
+* [Retrying data for jobs](/docs/apis/rest-api/jobs#retry-a-job).
+* [Getting a build by ID](/docs/apis/rest-api/builds#get-a-build).
+
+## Exclusive GraphQL features
```
Comment (plaindocs) [inline pages/api_differences.md.erb:21]: and 

> ## Features only available in the GraphQL API

File: pages/api_differences.md.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* [Increasing the next build number](/docs/apis/graphql/graphql-cookbook#increase-the-next-build-number).
+* [Getting all jobs in a given queue for a given timeframe](/docs/apis/graphql/graphql-cookbook#get-all-jobs-in-a-given-queue-for-a-given-timeframe).
+* [Creating a user into a specific team and permissions set](/docs/apis/graphql/graphql-cookbook#create-a-user-add-them-to-a-team-and-set-user-permissions).
+* [Getting all environment variables set on a build](/docs/apis/graphql/graphql-cookbook#get-all-environment-variables-set-on-a-build).
+* Listing audit log events.
+* Filtering results from pipeline listings.
+
+## Known missing API features
+
+These are know requested features that are currently missing from both REST and GraphQL APIs. We are working on bringing them to the list of available API features:
```
Comment (plaindocs) [inline pages/api_differences.md.erb:43]:
```suggestion
These are known requested features that are currently missing from both REST and GraphQL APIs:

```

This made me LOL

File: pages/api_differences.md.erb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+{:toc}
+
+If you are only deciding which approach to take, you could consider GraphQL as your default answer to "how do I query a possibly-complicatedly-constrained subset of data?", and REST should be most helpful in a scenario of creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Exclusive REST features
+
+* Buildkite's terraform provider uses GraphQL where it can, but it falls back to using REST for setting [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
```
Comment (plaindocs) [inline pages/api_differences.md.erb:13]:
```suggestion
* Setting [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
```

Comment (plaindocs) [reply pages/api_differences.md.erb:13]: this feels more consistent and findable. If the Terraform is important we can add it in after the main sentence?

File: pages/api_differences.md.erb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+* REST API can [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [`buildkite-agent annotate` command](/docs/agent/v3/cli-pipeline) from within a job).
```
Comment (plaindocs) [inline pages/api_differences.md.erb:14]:
```suggestion
*  [Retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [`buildkite-agent annotate` command](/docs/agent/v3/cli-pipeline) from within a job).
```

I'm not clear if this is an only REST feature, or if the important information here is that you can't create Annoitations

File: pages/api_differences.md.erb [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+* [Provider setting properties](/docs/apis/rest-api/pipelines#provider-settings-properties) `provider_settings` allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on `pipeline create`.
```
Comment (plaindocs) [inline pages/api_differences.md.erb:15]: I don't understand this.

File: pages/api_differences.md.erb [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Exclusive GraphQL features
+
+* [Removing users from an organization](/docs/apis/graphql/graphql-cookbook#delete-an-organization-member).
+* [Setting up SSO](/docs/integrations/sso/sso-setup-with-graphql).
+* [Retrieving, listing, and creating annotations on a build](/docs/apis/rest-api/annotations#Notification).
+* [Getting all the pipeline metrics from the dashboard from the API](/docs/apis/graphql/graphql-cookbook#get-pipeline-metrics).
+* [Creating and revoking](/docs/apis/rest-api/access-token) user API tokens (while REST can only revoke user API tokens).
+* Creating and managing pipeline schedules.
+* Managing teams ([add](/docs/apis/graphql/graphql-cookbook#add-an-existing-organization-user-to-a-team) or [remove](/docs/apis/graphql/graphql-cookbook#remove-a-team-member)a team member], [removing pipeline edit access from existing teams](/docs/apis/graphql/graphql-cookbook#set-teams-pipeline-edit-access-to-read-only-or-build-and-read).
```
Comment (plaindocs) [inline pages/api_differences.md.erb:29]:
```suggestion
* Managing teams ([add](/docs/apis/graphql/graphql-cookbook#add-an-existing-organization-user-to-a-team) or [remove](/docs/apis/graphql/graphql-cookbook#remove-a-team-member) a team member], [removing pipeline edit access from existing teams](/docs/apis/graphql/graphql-cookbook#set-teams-pipeline-edit-access-to-read-only-or-build-and-read).
```

**Review by plaindocs — COMMENTED** [review]
This is great!! I'd love to merge this after the suggested tweaks. 

Instead of duplicating the info, how about adding a small paragraph in the overview of each API linking to the limitations page? 

What do you think of tagging the entries according to REST endpoint (because the GraphQL is more monolithic), for example:

```
* BUILDS [setting [branch_configuration and github provider configuration](https://bk-docs-pr-1613.onrender.com/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
* ANNOTATIONS [retrieve and list annotations](https://bk-docs-pr-1613.onrender.com/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [buildkite-agent annotate command](https://bk-docs-pr-1613.onrender.com/docs/agent/v3/cli-pipeline) from within a job).
* PIPELINES [Provider setting properties](https://bk-docs-pr-1613.onrender.com/docs/apis/rest-api/pipelines#provider-settings-properties) provider_settings allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on pipeline create.
* ...
* 
```
and use the same labelling for the GrpahQL API?

File: pages/api_differences.md.erb (hunk 3 above)
Comment (plaindocs) [reply pages/api_differences.md.erb:13]: These links should be to the relevant API section, not to the generic docs. If it isn't clear what those are we'll need to do a bit more work.

File: pages/api_differences.md.erb [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+Buildkite supports the use of both [REST](https://buildkite.com/docs/apis/rest-api) and [GraphQL](/docs/apis/graphql-api), but there is no full parity in the API between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+If you are only deciding which approach to take, you could consider GraphQL as your default answer to "how do I query a possibly-complicatedly-constrained subset of data?", and REST should be most helpful in a scenario of creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Exclusive REST features
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/api_differences.md.erb:11]: This is better than my 2 initial variants, I like it.

File: app/views/layouts/_sidebar.html.erb [hunk 8]
```diff
@@ -257,6 +257,7 @@
       <div class="Docs__nav__section-content">
         <ul class="Docs__nav__sub-nav">
           <%= sidebar_link_to "Managing API Tokens", 'apis/managing-api-tokens' %>
+          <%= sidebar_link_to "API Disparity", 'apis/api-differences' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:260]: This doesn't seem to work? ?

Comment (karensawrey) [reply app/views/layouts/_sidebar.html.erb:260]: Oops. Moved the file to a proper place. Thanks!

File: pages/api_differences.md.erb [hunk 9]
```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity in the API between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
```
Comment (plaindocs) [inline pages/api_differences.md.erb]:
```suggestion
Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
```

**Comment by plaindocs** [comment]
I reckon the 'tags' are too bold, what do you think @karensawrey? 
And it makes sense to tag the missing features as well I think.

**Comment by plaindocs** [comment]
@olyism reckon you could whip up a `<%= api_label_pill('Users') %>` or something for these labels? https://bk-docs-pr-1613.onrender.com/docs/apis/api-differences#features-only-available-in-the-rest-api

Although maybe it doesn't need to be erb? But I definitely don't want a bunch of html in there.

File: app/views/layouts/_sidebar.html.erb (hunk 8 above)
Comment (sj26) [inline app/views/layouts/_sidebar.html.erb:260]:
```suggestion
          <%= sidebar_link_to "API Differences", 'apis/api-differences' %>
```

File: pages/apis/api_differences.md.erb [hunk 10]
```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:3]:
```suggestion
Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there are some differences between the two. Some tasks can only be achieved using the REST API, or the GraphQL API. We recommend using a mixture of both when required.
```

File: pages/apis/api_differences.md.erb [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
+
+* **ANNOTATIONS:** [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [buildkite-agent annotate command](/docs/agent/v3/cli-pipeline) from within a job).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:13]: Annotations are available in graphql. That’s how we render them on the build page:

https://github.com/buildkite/buildkite/blob/15812ebe5aabfeecda41a9fe125604b7420d6ac9/app/models/api/graph/types/build_type.rb#L218

File: pages/apis/api_differences.md.erb [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+* **BUILDS:** set [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:14]: Branch configuration can be read and updated on a pipeline via the graphql api:

https://github.com/buildkite/buildkite/blob/15812ebe5aabfeecda41a9fe125604b7420d6ac9/app/models/api/graph/types/pipeline_type.rb#L76

File: pages/apis/api_differences.md.erb [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+* **BUILDS:** [get a build by ID](/docs/apis/rest-api/builds#get-a-build).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:15]: A build can be looked up via uuid in graphql:

https://github.com/buildkite/buildkite/blob/15812ebe5aabfeecda41a9fe125604b7420d6ac9/app/models/api/graph/types/query_type.rb#L65

File: pages/apis/api_differences.md.erb (hunk 12 above)
Comment (sj26) [reply pages/apis/api_differences.md.erb:14]: Looks like github provider configuration is covered by pipelines below?

File: pages/apis/api_differences.md.erb [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+* **JOBS:** [get an output of job logs](/docs/apis/rest-api/jobs#get-a-jobs-log-output).
+* **JOBS:** [retry data for jobs](/docs/apis/rest-api/jobs#retry-a-job).
+* **META:** [get a list of IPs from which Buildkite sends webhooks](/docs/apis/rest-api/meta#get-meta-information).
+* **PIPELINES:** [set provider properties](https://bk-docs-pr-1613.onrender.com/docs/apis/rest-api/pipelines#provider-settings-properties) `provider_settings` allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on pipeline create.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:19]: oops, looks like a render url snuck in?

File: pages/apis/api_differences.md.erb [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+## Features only available in the GraphQL API
+
+* **AGENTS:** [get a list of agent token IDs](/docs/apis/graphql/graphql-cookbook#get-a-list-of-agent-token-ids).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:23]: “Agent tokens” is generally only available via graphql. (Agent token improvements is on the roadmap, and will likely include some work on the APIs.)

File: pages/apis/api_differences.md.erb [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+* **BUILDS:** [get all environment variables set on a build](/docs/apis/graphql/graphql-cookbook#get-all-environment-variables-set-on-a-build).
+* **BUILDS:** [increase the next build number](/docs/apis/graphql/graphql-cookbook#increase-the-next-build-number).
+* **BUILDS:** [retrieve, list, and create annotations on a build](/docs/apis/rest-api/annotations).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:26]:
```suggestion
* **BUILDS:** [create annotations on a build](/docs/apis/rest-api/annotations).
```

File: pages/apis/api_differences.md.erb [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+* **JOBS:** [get all jobs in a given queue for a given timeframe](/docs/apis/graphql/graphql-cookbook#get-all-jobs-in-a-given-queue-for-a-given-timeframe).
+* **JOBS:** list audit log events.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:28]:
```suggestion
* **JOBS:** list job events.
```

File: pages/apis/api_differences.md.erb [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+* **ORGANIZATIONS:** [remove users from an organization](/docs/apis/graphql/graphql-cookbook#delete-an-organization-member).
+* **ORGANIZATIONS:** [set up SSO](/docs/integrations/sso/sso-setup-with-graphql).
+* **ORGANIZATIONS:** manage teams - [add](/docs/apis/graphql/graphql-cookbook#add-an-existing-organization-user-to-a-team) or [remove](/docs/apis/graphql/graphql-cookbook#remove-a-team-member) a team member.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:31]: Teams in general is mostly graphql only. The rest api can list teams, and can create pipelines in teams. Other team management is graphql only.

File: pages/apis/api_differences.md.erb [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+* **PIPELINES:** filter results from pipeline listings.
+* **PIPELINES:** create and manage pipeline schedules.
+* **USERS:** [create and revoke](/docs/apis/rest-api/access-token) user API tokens (while REST can only revoke user API tokens).
+* **USERS:** [create a user into a specific team and permissions set](/docs/apis/graphql/graphql-cookbook#create-a-user-add-them-to-a-team-and-set-user-permissions).
+
+## Known missing API features
+
+These are known requested features that are currently missing from both REST and GraphQL APIs:
+
+* **PIPELINES:** create notification settings.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:46]:
```suggestion
* **NOTIFICATION SERVICES:** there is no API for managing notification services.
```

File: pages/apis/api_differences.md.erb [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+* **USERS:** display secondary user email.
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:47]:
```suggestion
* **USERS:** display secondary user emails.
```

**Comment by sj26** [comment]
It might be useful context to point out that the REST API is a good choice for org level stuff, but wasn’t built for user tasks. For example, it wasn’t intended for api token or email management on a user. However GraphQL is more comprehensive, and often can achieve things a single user would want to do in the UI.

File: pages/apis/api_differences.md.erb [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:11]: The REST API also exclusively allows showing some information about the access token currently in use, and revoking the current access token:

https://buildkite.com/docs/apis/rest-api/access-token#get-the-current-token

File: pages/apis/api_differences.md.erb [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+
+## Features only available in the GraphQL API
+
+* **AGENTS:** [get a list of agent token IDs](/docs/apis/graphql/graphql-cookbook#get-a-list-of-agent-token-ids).
+* **BUILDS:** [get all environment variables set on a build](/docs/apis/graphql/graphql-cookbook#get-all-environment-variables-set-on-a-build).
+* **BUILDS:** [increase the next build number](/docs/apis/graphql/graphql-cookbook#increase-the-next-build-number).
+* **BUILDS:** [retrieve, list, and create annotations on a build](/docs/apis/rest-api/annotations).
+* **JOBS:** [get all jobs in a given queue for a given timeframe](/docs/apis/graphql/graphql-cookbook#get-all-jobs-in-a-given-queue-for-a-given-timeframe).
+* **JOBS:** list audit log events.
+* **ORGANIZATIONS:** [remove users from an organization](/docs/apis/graphql/graphql-cookbook#delete-an-organization-member).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:29]: I think org/user membership management is graphql exclusive almost entirely, including teams and SSO as you've got below.

File: app/views/layouts/_sidebar.html.erb (hunk 8 above)
Comment (plaindocs) [reply app/views/layouts/_sidebar.html.erb:260]: Oh yeah, I meant to suggest this too. :+1:

File: pages/apis/api_differences.md.erb (hunk 14 above)
Comment (karensawrey) [reply pages/apis/api_differences.md.erb:19]: yeah! Thanks for catching it!

**Review by plaindocs — APPROVED** [review]
Such commit history!!

The only thing I'd add here is a small sentence in /docs/apis/rest-api and /docs/apis/graphql pointing to this difference list.

**Comment by plaindocs** [comment]
> It might be useful context to point out that the REST API is a good choice for org level stuff, but wasn’t built for user tasks. For example, it wasn’t intended for api token or email management on a user. However GraphQL is more comprehensive, and often can achieve things a single user would want to do in the UI.

@sj26 I'd argue that folks are less likely to care what the original intent was, and more about the current state.

**Comment by karensawrey** [comment]
@plaindocs indeed! Onto adding that link to those pages (once I've resolved the merge conflicts... sigh...)! 
As for 
>folks are less likely to care what the original intent was, and more about the current state.
current state is what I'd taken and integrated from @sj26's comment 🤘

File: pages/apis/graphql_api.md.erb [hunk 23]
```diff
@@ -1,6 +1,8 @@
 # GraphQL API Overview
 
-The Buildkite GraphQL API provides an alternative to the [REST API](/docs/apis/rest-api). It allows for more efficient retrieval of data by enabling you to fetch multiple, nested resources in a single request. The Buildkite GraphQL API is currently in beta.
+The Buildkite GraphQL API provides an alternative to the [REST API](/docs/apis/rest-api). It allows for more efficient retrieval of data by enabling you to fetch multiple, nested resources in a single request.
+
+For the list of existing disparities between the REST API and the GraphQL API, see [API Differences](/docs/api-differences).
```
Comment (plaindocs) [inline pages/apis/graphql_api.md.erb:5]:
```suggestion
For the list of existing disparities between the REST API and the GraphQL API, see [API Differences](/docs/apis/api-differences).
```

**Comment by olyism** [comment]
> @olyism reckon you could whip up a <%= api_label_pill('Users') %> or something for these labels? https://bk-docs-pr-1613.onrender.com/docs/apis/api-differences#features-only-available-in-the-rest-api

@plaindocs I've pushed up 2 commits with the pill helper methods 653edb7d1efb497300c905c9d71a368ba67bfbab and putting them into the page a4beb3b4f86f892b3713e17b5399de20a35f4691

The pill SCSS file is getting a bit cumbersome listing out each pill, but will have a follow up ticket later to refactor this.

**Comment by plaindocs** [comment]
Speedy work @olyism . We can re-examine that approach, sure. Just waiting for Render to deploy that change.

@karensawrey looks like you added the diff link to only one API?

**Comment by karensawrey** [comment]
@plaindocs this is funny because I actually added the link to the REST docs first, but didn't save the file :D Adding that commit now.

**Comment by plaindocs** [comment]
Squash it and ship it!

File: pages/apis/api_differences.md.erb [hunk 24]
```diff
@@ -0,0 +1,47 @@
...
+* <%= pill "PIPELINES", "pipelines" %> [get the last build's creation date within every pipeline](/docs/apis/graphql/graphql-cookbook#get-the-creation-date-of-the-most-recent-build-in-every-pipeline).
+* <%= pill "PIPELINES", "pipelines" %> [count the number of builds on a branch](/docs/apis/graphql/graphql-cookbook#count-the-number-of-builds-on-a-branch).
+* <%= pill "PIPELINES", "pipelines" %> [get the creation date of the most recent build in every pipeline](/docs/apis/graphql/graphql-cookbook#get-the-creation-date-of-the-most-recent-build-in-every-pipeline).
+* <%= pill "PIPELINES", "pipelines" %> filter results from pipeline listings.
+* <%= pill "PIPELINES", "pipelines" %> create and manage pipeline schedules.
+* <%= pill "USERS", "users" %> [create and revoke](/docs/apis/rest-api/access-token) user API tokens (while REST can only revoke user API tokens).
+* <%= pill "USERS", "users" %> [create a user into a specific team and permissions set](/docs/apis/graphql/graphql-cookbook#create-a-user-add-them-to-a-team-and-set-user-permissions).
```
Comment (sj26) [inline pages/apis/api_differences.md.erb:39]: Slight wording thing — organizations don't create users, they invite users.

Organization membership management, is exclusive to GraphQL, including inviting users to become members with specific teams or with a specific role.

Comment (karensawrey) [reply pages/apis/api_differences.md.erb:39]: Thanks for the comment, @sj26! I'll push this update and another update already on my mind to this PR during the EU hours today. Highly appreciated! <3

Comment (sj26) [reply pages/apis/api_differences.md.erb:39]: I appreciate all the work you're putting into this! 💚

**Pull request merged**